### PR TITLE
EDSC-3081: Updates subscriptions query stringification

### PR DESCRIPTION
**Pull request by macrouch**

### CODE REVIEW

**Review by abbottry — APPROVED** [review]

File: static/src/js/util/subscriptions.js [hunk 1]
```diff
@@ -16,5 +16,5 @@ export const prepareSubscriptionQuery = (params) => {
   delete prunedParams.sortKey
 
   // Return the remaining parameters stringified, don't encode the values because CMR won't accept them
-  return stringify(prunedParams, { encode: false })
+  return stringify(prunedParams, { encode: false, indices: false, arrayFormat: 'brackets' })
```
Comment (ygliuvt) [inline static/src/js/util/subscriptions.js:19]: There are some granule search parameters in CMR that are indexed. e.g. https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html#g-passes.

Comment (ygliuvt) [reply static/src/js/util/subscriptions.js:19]: Will this code handle the indexed case?

**Review by trevorlang — APPROVED** [review]

File: static/src/js/util/subscriptions.js (hunk 1 above)
Comment (macrouch) [reply static/src/js/util/subscriptions.js:19]: We don't support those parameters, do you know of any others?

Comment (ygliuvt) [reply static/src/js/util/subscriptions.js:19]: Good. That is only one on granules. But if this function would also be used for collection searches, you are in trouble as science-keywords parameter are indexed.

Comment (macrouch) [reply static/src/js/util/subscriptions.js:19]: Cool, this code is only for subscriptions, so only granule queries

Comment (ygliuvt) [reply static/src/js/util/subscriptions.js:19]: Based on where this code is located, I don't think it is used for collection searches.

**Pull request merged**